### PR TITLE
[DOCFIX] Add more details to Docker doc

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

File: docs/en/Running-Alluxio-On-Docker.md [hunk 1]
```diff
@@ -43,15 +44,30 @@ $ docker build -t alluxio .
...
 ```bash
 $ docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
 ```
 
+### Set up under storage
+
+To keep this guide simple, we will use the local filesystem for under storage. In a real deployment
+you would instead use something like HDFS or S3.
+
+Create an under storage folder
```
Comment (gpang) [inline docs/en/Running-Alluxio-On-Docker.md]: on the host?

File: docs/en/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -43,15 +44,30 @@ $ docker build -t alluxio .
...
+```bash
+$ mkdir underStorage
```
Comment (gpang) [inline docs/en/Running-Alluxio-On-Docker.md]: use `/path/to/underStorage` like below?

File: docs/en/Running-Alluxio-On-Docker.md [hunk 3]
```diff
@@ -84,12 +113,14 @@ This way, the data written by the Alluxio worker can be directly accessed from o
...
-           -e ALLUXIO_WORKER_MEMORY_SIZE=10GB \
-           --net=host alluxio worker
+$ docker run -d --net=host \
+             -v /mnt/ramdisk:/mnt/ramdisk \
+             -v $PWD/underStorage:/underStorage \
+             -e ALLUXIO_MASTER_HOSTNAME=${INSTANCE_PUBLIC_IP} \
+             -e ALLUXIO_RAM_FOLDER=/mnt/ramdisk \
+             -e ALLUXIO_WORKER_MEMORY_SIZE=1GB \
+             -e ALLUXIO_UNDERFS_ADDRESS=/underStorage \
+             aee worker
```
Comment (gpang) [inline docs/en/Running-Alluxio-On-Docker.md]: aee?

File: docs/en/Running-Alluxio-On-Docker.md [hunk 4]
```diff
@@ -123,21 +154,34 @@ and replacing periods with underscores. For example, `alluxio.master.hostname` c
 when the image starts.
 
 ```bash
-$ docker run -d --net=host -e ALLUXIO_MASTER_HOSTNAME=ec2-203-0-113-25.compute-1.amazonaws.com alluxio worker
+$ docker run -d --net=host \
+             -v $PWD/underStorage:/underStorage \
+             -e ALLUXIO_MASTER_HOSTNAME=${INSTANCE_PUBLIC_IP} \
+             -e ALLUXIO_UNDERFS_ADDRESS=/underStorage \
+             aee worker
```
Comment (gpang) [inline docs/en/Running-Alluxio-On-Docker.md]: aee?

File: docs/en/Running-Alluxio-On-Docker.md [hunk 5]
```diff
@@ -123,21 +154,34 @@ and replacing periods with underscores. For example, `alluxio.master.hostname` c
...
-$ docker run -d --net=host --shm-size=50GB \
-           -e ALLUXIO_MASTER_HOSTNAME=master \
-           -e ALLUXIO_WORKER_MEMORY_SIZE=50GB \
-           alluxio worker
+$ docker run -d --net=host --shm-size=1G \
+             -v $PWD/underStorage:/underStorage \
+             -e ALLUXIO_MASTER_HOSTNAME=${INSTANCE_PUBLIC_IP} \
+             -e ALLUXIO_WORKER_MEMORY_SIZE=1GB \
+             -e ALLUXIO_UNDERFS_ADDRESS=/underStorage \
+             aee worker
```
Comment (gpang) [inline docs/en/Running-Alluxio-On-Docker.md]: aee?

File: docs/en/Running-Alluxio-On-Docker.md (hunk 1 above)
Comment (aaudiber) [inline docs/en/Running-Alluxio-On-Docker.md]: done

File: docs/en/Running-Alluxio-On-Docker.md (hunk 2 above)
Comment (aaudiber) [inline docs/en/Running-Alluxio-On-Docker.md]: changed to use `/underStorage` everywhere

**Comment by aaudiber** [comment]
@gpang Thanks for the review, I've addressed the comments

**Pull request merged**